docs/tutorial/shell.md

```diff
@@ -0,0 +1,65 @@
+## Run Spark Interactively
+
+Apache Spark comes with two shells to be used in interactive mode:
+* `spark-shell`, that is built on top of the Scala REPL shell
+* `pyspark`, that is built on top of the python interpreter shell
+
+### Interactive Spark Shell
+For interactive use cases, spark-client snap ships with Apache Spark's spark-shell utility.
+
+It's a useful tool to quickly validate your assumptions about Spark in Scala before finding out after an actual long running job failure.
+
+Great! Let us test out our spark-shell setup with the official Pi example from the Spark distribution.
+
+```shell
+$ spark-client.spark-shell
+....
+....
+Spark session available as 'spark'.
+Welcome to
+      ____              __
```

Comment:
who wrote this example code and if not us, what is the license for it?

Reply:
well, this is JUST the example of the output of the shell. Do we need to have a license for this?

Reply:
reworded.